### PR TITLE
feat(colors): dark colors list containers and separator

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Enhancement

#### The following changes where made:

As part of the migration of new colors from design, I changed background and border colors of list, containers and separator. In light mode it doesn't change anything as background & backgroundWeak are same but in dark mode it add more contrast.

## Relevant logs and/or screenshots

| Page |   Before   |      After |
| :--- | :--------: | ---------: |
| instance > overview  | Screenshot 2022-04-07 at 15 35 03 | Screenshot 2022-04-07 at 15 33 47 |
| instance > listing  | Screenshot 2022-04-07 at 15 37 26 | Screenshot 2022-04-07 at 15 36 40 |

### CODE REVIEW

File: scripts/zIndex.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (chambo-e) [inline scripts/zIndex.json:1]: 6f4

Comment (matthprost) [reply scripts/zIndex.json:1]: oups that shouldn't be in that PR haha

Comment (matthprost) [reply scripts/zIndex.json:1]: oups that shouldn't be in that PR haha

**Comment by matthprost** [comment]
> @matthprost
> 
> Tooltip NOK Screenshot 2022-04-07 at 16 14 14
> 
> Tooltip NOK Screenshot 2022-04-07 at 16 14 22 Bout de fleche NOK Screenshot 2022-04-07 at 16 19 47 Mauvais violet sur le tick Screenshot 2022-04-07 at 16 20 51 text invisible Screenshot 2022-04-07 at 16 22 05 Screenshot 2022-04-07 at 16 22 30 Screenshot 2022-04-07 at 16 22 43 Screenshot 2022-04-07 at 16 22 37 Screenshot 2022-04-07 at 16 23 57 Screenshot 2022-04-07 at 16 25 36 Screenshot 2022-04-07 at 16 26 05

@vincentaudebert fixes here: https://github.com/scaleway/scaleway-ui/pull/1355

**Review by philibea — APPROVED** [review]

**Review by vincentaudebert — APPROVED** [review]

**Pull request merged**